projectZoo/src/pr1/Main.java

```diff
@@ -1,7 +1,91 @@
-package pr1;
+package pr1; 
 
-public class Main {
-    public static void main(String args[]){
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.Scanner;
 
+public class Main{
+    Scanner sc = new Scanner(System.in);
+
+    static RandomQuantity random = new RandomQuantity();
+    static int randomQuant = random.random();
+
+    static String[] names = {"Shapito", "Karamel", "Cucumber", "Dolce i Kaban", "Winni Duck"};
+    static String[] locations = {"l`viv", "Taraso-Shevchenkivsk", "Old York", "Riznutsi", "Odessa-dad", "Pryamovo"};
+    static Zoo first = new Zoo(names, locations, 50, randomQuant);
+
+    static int answer = -1;
+//_______________________________________________________________________
+    public static void main(String[] args) throws InterruptedException, IOException, ParseException {
+        Event e = new Event("First stage", "Outside", "Recruiting people");
+
+        e.theFirstStage();
+    }
+ 
+    protected void menu(){
+ 
+        System.out.println("You`re calling to Zoo. What do you want to find out or do?"); 
+        System.out.println("Number of tickets left[1]; Ticket price[2]; Name[3]; Locations[4]; Buy a ticket[5]"); 
+ 
+        exepts(); 
+ 
+    } 
+ 
+    void info() throws ParseException {
+        String a = sc.nextLine(); 
+        answer = Integer.parseInt(a); 
+        if (answer > 5 | answer < 1) { 
```

Comment:
Magic Number, мені як сторонньому програмісту який дивиться лише на код тяжко зрозуміти що за числа 5 та 1, пробуйте в такому випадку використовувати константи, навіть створити окремий клас констант(static).

Reply:
Встановив межі допустимих значень для відповіді:
```
    static final int minAnswerLimit = 1;
    static final int maxAnswerLimit = 5;
```
Тоді, по прикладу нижче можна зробити висновок, що щось повинно відбутись, якщо answer буде між мінімальним значенням та максимальним.
```suggestion
      if (answer > maxAnswerLimit | answer < minAnswerLimit) {
            exceptToString();
```